sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -285,15 +286,15 @@ def __init__(self, command, prefix=None, continuation_token=None):
         self._headers = None
         self.location_mode = None
 
-    def _get_next_cb(self, continuation_token):
+    def _get_next_cb(self, continuation_token, **kwargs):
         try:
             return self._command(
                 next_table_name=continuation_token or None,
-                cls=return_context_and_deserialized,
+                cls=kwargs.pop('cls', _return_context_and_deserialized),
```

Comment:
I tend to prefer `kwargs.pop('cls', None) or _return_context_and_deserialized` just in case someone passes in `cls=None`